### PR TITLE
Setup and controllers

### PR DESCRIPTION
**Pull request by awais-amjed**
In this Pull Request, the following changes were introduced:

- Created a new rails application ✅ 
- Added database connection parameters ✅ 
- Added Linters configuration ✅ 
- Added Controllers, Models, routes, and Views for Posts and Users ✅

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:id", to: "posts#show"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
```
Comment (ShoiraTa) [inline config/routes.rb]: - [ ] OPTIONAL: You can use resources to define the routes:

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @awais-amjed ,

Well done: 
- Has `users_controller` and `posts_controller` with `#index` and `#show` actions ✔️ 
- Has corresponding views and routes ✔️ 

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.


There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

- [ ] Kindly update the readme file, looks like you were so exited with the project that forgot about the readme this time 🙈 

## Optional suggestions

- [ ] See comments below

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (awais-amjed) [reply config/routes.rb]: Thanks, I have updated it.

**Review by DeliceLydia — APPROVED** [review]
## Hi @awais-amjed, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**